spec/controllers/games_controller_spec.rb

```diff
@@ -25,4 +25,25 @@
       expect(response).to have_http_status(:success)
     end
   end
+
+  describe 'games#index action' do
+    it "should succesfully show the page" do
+      get :index
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'should give an overview of games with only one player' do
+      game1 = FactoryGirl.create(:game)
+      game2 = FactoryGirl.create(:game)
+      game3 = FactoryGirl.create(:game)
+      player1 = FactoryGirl.create(:player, game_id: game1.id)
+      player2 = FactoryGirl.create(:player, game_id: game2.id)
```

Comment:
Yup it's just saying that you're not using the variables, which is common for a linter - it means you're wasting memory. 

It's standard practice to just delete the variables that are not used. TBH the naming is irrelevant here anyways right? 

If a developer were to look at the code, they would be able to ascertain that one game doesn't have a user, one has one, and one has two. Think of a word problem back in Math class:

```
Mike has five apples, he sells two apples to Mary & one apple to Joe, how many apples does Mike have left?
```

In this example, Mike is `game1`. Mary and Joe are 2 & 3, which are irrelevant to the problem at hand - other than the fact that they took apples.

Make sense?